packages/config/src/layer2s/zkspace.ts

```diff
@@ -117,6 +118,26 @@ export const zkspace: Layer2 = {
     destinationToken: RISK_VIEW.NATIVE_AND_CANONICAL(),
     validatedBy: RISK_VIEW.VALIDATED_BY_ETHEREUM,
   }),
+  stage: getStage({
+    stage0: {
+      callsItselfRollup: true,
+      stateRootsPostedToL1: true,
+      dataAvailabilityOnL1: true,
+      rollupNodeOpenSource: ['UnderReview', ''],
```

Comment:
What is being shown on the UI in this case?

Reply:
for now nothing, in the next PR design will be implemented